src/utilities.cc

```diff
@@ -299,8 +305,26 @@ static void MyUserNameInitializer() {
   if (user != NULL) {
     g_my_user_name = user;
   } else {
-    g_my_user_name = "invalid-user";
+#if defined(HAVE_PWD_H) && defined(HAVE_UNISTD_H)
+    uid_t uid;
+    struct passwd pwd;
+    struct passwd*result = NULL;
```

Comment:
Could you add a whitespace right before "result"?

struct passwd* result = NULL